src/offscreen/offscreen.ts

```diff
@@ -0,0 +1,42 @@
+import { config } from 'config'
+import { ConnectorClient } from 'connector/connector-client'
+import { ledgerResponseDiscriminator } from 'ledger/schemas'
+import { logger } from 'utils/logger'
+
+const connectorClient = ConnectorClient({
+  source: 'extension',
+  target: 'wallet',
+  signalingServerBaseUrl: config.signalingServer.baseUrl,
+  isInitiator: true,
+  logger,
+})
+
+connectorClient.onMessage$.subscribe(chrome.runtime.sendMessage)
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+  interface Window {
+    sendWebRTCMessage: typeof connectorClient.sendMessage
+  }
+}
+
+window.sendWebRTCMessage = connectorClient.sendMessage
```

Comment:
why do we want to add this to the global scope?

Reply:
reverted